src/odemis/gui/cont/stream_bar.py

```diff
@@ -1198,6 +1199,42 @@ class SecomStreamsController(StreamBarController):
     Controls the streams for the SECOM and DELPHI live view
     """
 
+    def _acqAddFluo(self, **kwargs):
+        """
+        Set the Fluostream attributes to the selected acquired stream panel setting
+        """
+        name = kwargs["label"]
+        s = acqstream.FluoStream(
+            name,
+            self._main_data_model.ccd,
+            self._main_data_model.ccd.data,
+            self._main_data_model.light,
+            self._main_data_model.light_filter,
+            focuser=self._main_data_model.focus,
+            opm=self._main_data_model.opm,
+            detvas={"exposureTime"},
+        )
+        index = next((i for i, k in enumerate(self.acq_settings.config_data) if k["name"] == name), None)
```

Comment:
This iterator is kind of hard to read, I would recommend replacing it with a helper function, as it is used multiple times, see previous comment